apps/quick-learn-frontend/src/app/(Dashboard)/dashboard/learning-path/Dashboard.tsx

```diff
@@ -0,0 +1,171 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import ProgressCard from '@src/shared/components/ProgressCard';
+import { TUserCourse, TUserRoadmap } from '@src/shared/types/contentRepository';
+import { getUserRoadmapsService } from '@src/apiServices/contentRepositoryService';
+import { en } from '@src/constants/lang/en';
+import CardSkeleton from '@src/shared/components/CardSkeleton';
+
+const DashboardSkeleton = () => (
```

Comment:
This component will be used only here, but it would be better to place it separately.